AUDITING.md

```diff
@@ -0,0 +1,75 @@
+# Auditing
+
+This document defines how Platform Chaos audits chaos events. It lists what is tracked and how it is formatted.
+
+## Tracked Data
+
+When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the following properties are recorded and exported through a JSON object.
+  - An `id` unique to this event instantiation (`guid`). 
+  - A `chaos-type` of enumerable type `initiate`
+  - `user` and `system` that terminated the event
+  - `date` of event initiation (including time `UTC`)
+  - targeted `resources`. This is broken down by `subscriptionID` and then by `resourceGroup`
+  - Chaos `event` information (`eventID`, `eventName`, `eventURL`)
+
+Additionally an *Extension* may record additional information regarding the specificities of the event. This information is stored within an `extension` property.
+
+Every chaos event initiation/termination will be logged. The general structure of an event log is defined below:
+
+- _event log_ (Object)
+  - `id` (String) 
+    - GUID unique to this event instantiation
+  - `type` (String) 
+    - One of `'initiate'`, `'terminate'`
+  - `user` (String)
+    - Some identifiable piece of information specifying which user executed the chaos event
+  - `system` (String | Object)
+    - Some identifiable piece of information specifying which system the chaos event was executed from
```

Comment:
this is not specific enough.

Reply:
This is something i wanted to discuss.

1. Is this important information to track
2. How do we track this kind of info easily
3. Some ideas I had for object attributes would inlcude: `OS`, `Runtime`, `Location`, `IP Address`

Let me know your thoughts on those points please ^

Reply:
gotcha. yes given that data, I actually don't think this is super valuable...and given the complexity of trying to obtain this data across the potential different machines extensions can run on, its probably worth punting at this point.